source/sync/data-model/data-model-map.txt

```diff
@@ -45,6 +45,10 @@ A conventional approach is to name each object type with a singular
 noun, like "Dog" or "Person". Schemas generated in development mode or
 by sampling existing documents use this convention.
 
+.. note:: 
+
+   To work with Atlas Device Sync, type names cannot exceed 57 UTF-8 characters.
```

Comment:
It might be a good idea to create a source constant (in snooty.toml) for the number of characters, since we're repeating it in multiple places (plus on the Realm side, though at least there it's contained in a single note file that gets included everywhere). After adding the source constant to snooty.toml, the usage would be like `{+whatever-constant-name+} UTF-8 characters.`

Reply:
Agreed. Done 👍